client/src/main/java/com/paypal/selion/configuration/Config.java

```diff
@@ -526,6 +541,12 @@ public static synchronized void setConfigProperty(ConfigProperty configProperty,
         SITE_LOCALE("siteLocale", "US", false),
 
         /**
+         * Site will show country used for tests.<br>
```

Comment:
The `MOBILE_PLATFORM` is almost the same as `ConfigProperty.BROWSER_CAPABILITY_PLATFORM`, but `BROWSER_CAPABILITY_PLATFORM` does not make any sense here, since it was designed for browser.

the same goes to `ConfigProperty.BROWSER_CAPABILITY_VERSION` (i did not create any new parameter for this one.)